device-types/Cisco/N9K-C9372TX-E.yaml

```diff
@@ -0,0 +1,127 @@
+---
+manufacturer: Cisco
+model: Nexus 9372TX-E
+part_number: N9K-C9372TX-E
+slug: n9k-C9372tx-e
+u_height: 1
+is_full_depth: false
```

Comment:
Please add a comment in markdown format with a link to the manufacturers datasheet, if you are able to find a publicly available one